src/Host/Client/Impl/Definitions/IRBlobHelper.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.R.Host.Client {
+    public interface IRBlobHelper {
```

Comment:
Will update in the next iteration.

---
In reply to: [70852735](https://github.com/Microsoft/RTVS/pull/2076#discussion_r70852735) [](ancestors = 70852735)